src/components/Route/CFunctionsTable.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div>
+    <table
+      class="my-3 w-100"
+    >
+      <th>
+        <span>Functions</span>
+        <b-dropdown
+          text="Add function"
+          class="ml-2 dropdown-function"
+          variant="light"
+        >
+          <b-dropdown-item
+            v-for="(func, index) in avaliableFunctions"
+            :key="index"
+            href="#"
+            @click="onAddFunction(func)"
+          >
+            {{ func.label }}
+          </b-dropdown-item>
+        </b-dropdown>
+      </th><th>
+        Status
+      </th>
+      <th class="text-center">
+        Actions
+      </th>
+      <template v-if="functions.length">
+        <tr
+          v-for="(func, index) in functions"
+          :key="index"
+          class="pointer"
+          :class="[selectedRow===index ? 'row-selected' : 'row-not-selected']"
+          active
+          @click="onRowClick(func,index)"
+        >
+          <td>
+            {{ func.label }}
+          </td>
+          <td>
+            Active
+          </td>
+          <td class="text-center">
+            <b-button
+              variant="danger"
+              class="my-1"
+              size="sm"
+              @click="onRemoveFunction(index)"
+            >
+              Remove
+            </b-button>
+          </td>
+        </tr>
+      </template>
+      <template v-else>
+        <tr class="mt-2 text-danger">
+          Please add a function!
+        </tr>
+      </template>
+    </table>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    avaliableFunctions: {
+      type: Array,
+      required: true,
+    },
+    functions: {
+      type: Array,
+      required: true,
+    },
+  },
+
+  data () {
+    return {
+      selectedRow: 0,
+    }
+  },
+
+  methods: {
+    onAddFunction (func) {
+      if (!this.functions.find(f => f.ref === func.ref)) {
+        this.functions.push({ ...func })
+      }
+      if (this.functions.length === 1) {
+        this.$emit('functionSelect', this.functions[0])
+      }
+    },
+    onRemoveFunction (index) {
+      this.functions.splice(index, 1)
+    },
+    onRowClick (func, index) {
+      this.selectedRow = index
+      this.$emit('functionSelect', func)
+    },
+  },
+}
+</script>
+
+<style scoped lang="scss">
+th{
```

Comment:
@MarioBur please handle this by using b-table Bootstrap Vue parameters available. for this concrete case, you have `th-class` parameter available and for borders you can add `border-top border-bottom`

Since you're adding the border to the `<td>` too, probably `bordered` for the whole table will be sufficient. In any case - make sure it looks consistent with all other tables (i.e. compose admin)

https://bootstrap-vue.org/docs/components/table#tables